app/src/main/java/com/github/sdp/ratemyepfl/activity/classrooms/RoomReviewsListActivity.kt

```diff
@@ -60,6 +63,14 @@ class RoomReviewsListActivity : AppCompatActivity() {
                 { fab.shrink() }
             )
         )
+
+        // Vertical swipe refreshes the list of reviews
+        swipeRefresher = findViewById(R.id.swiperefresh)
+        swipeRefresher.setOnRefreshListener {
+            viewModel.refreshReviewsList()
+            swipeRefresher.isRefreshing = false
```

Comment:
I don't really get why you need to set isRefreshing to false. I think I'll have a look at the documentation.